label_studio/core/all_urls.json

```diff
@@ -1013,6 +1013,12 @@
         "name": "ml_models:api:models-detail",
         "decorators": ""
     },
+    {
+        "url": "/api/models/compatible-projects",
```

Comment:
Not sure this is correct since the url is `/api/models/{id}/compatible-projects` right? Dont need to add this part manually there is a script that does it I will reference it here

Reply:
Patrick and I spoke yesterday, as per user flow we will get this api request during model creation and not after model is created so we will not have id info.

Reply:
In that case we should change the API accordingly to not accept ID, the url to not have the ID, etc

Reply:
yup changed that.